### ISSUE TITLE
--debug-html seems broken

### ISSUE THREAD

**Issue by glitsj16** [lwindolf/liferea#1092]
The `--debug-html` flag no longer produces detailed output. Regardless of turning 'Use Reader mode' on or off, the generated `output.html` always looks pretty bare:

```console
$ cat ~/.cache/liferea/output.html
<!DOCTYPE html>
<html>
<head>
<title>HTML View</title><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><base href="https://www.omgubuntu.co.uk" />
</head><body></body></html>
```

Is there any other way to get the rendered HTML for debugging?

<details>
<summary> System info </summary>

```
OS: Arch Linux
liferea-git 1.13.7.r84.gfa04083b-1
enabled plugins: Header Bar
internal browser settings: Never run external javascript
privacy settings:
    Use Reader mode
    Tell sites I do not want to be tracked
```

</details>

**Comment by mhoran** [comment]
Fix for this in #1093. Note that `output.html` won't show content modified by JavaScript (e.g. Reader Mode) but the inspector can be used in such cases. This fixes the issue where items are loaded but the content is not written to `output.html`.

**Comment by glitsj16** [comment]
@mhoran Just did a build from your fix-debug-html branch and can confirm the patch works great. Hope this gets merged soon. Thanks a bunch!
